client-core/src/wallet/syncer.rs

```diff
@@ -188,20 +232,21 @@ fn load_view_key<S: SecureStorage>(storage: &S, name: &str, enckey: &SecKey) ->
         })
 }
 
-impl<S, C, O, T> WalletSyncer<S, C, TxObfuscationDecryptor<O>, T>
+impl<S, C, O, T, L> WalletSyncer<S, C, TxObfuscationDecryptor<O>, T, L>
 where
     S: SecureStorage,
     C: Client,
     O: TransactionObfuscation,
     T: AddressRecovery,
+    L: Handle + Send + Sync + Clone,
```

Comment:
This `Handle + Send + Sync + Clone` is quite repeated, so maybe give a alias:
```
type LightClient = Handle + Send + Sync + Clone;
```
?

Reply:
done, made an extension trait for that.